File: Java/Array/Palindrome.java
public class Program {
	public static void main (String[] args)	{
        int vet[] = new int[50];
        for (int i = 0; i < vet.length; i++) {
            if (i % 2 == 0) {
                vet[i] = (i + 1) * 2;
            }
        }
        for (int i = 0; i < vet.length; i++) {
       	if (vet[i] == vet[vet.length - i - 1]) {
                System.out.println(vet[i] + " É PALINDROMO");
            }
        }
	}
}

//https://pt.stackoverflow.com/q/188867/101

File: CSharp/String/Palindrome.cs
using static System.Console;
					
public class Program {
	public static void Main() {
		WriteLine(EhPalindromo("4004"));
		WriteLine(EhPalindromo("4002"));
	}
	public static bool EhPalindromo(string texto) {
	    for (int i = texto.Length - 1; i > texto.Length / 2; i--) {
			if (texto[i] != texto[texto.Length - i - 1]) {
				return false;
			}
        }
		return true;
	}
}

//https://pt.stackoverflow.com/q/85688/101
